src/Service/Context/PageContext.php

```diff
@@ -94,7 +95,9 @@ public function __construct(ConfigFactoryInterface $config_factory, EntityTypeMa
    */
   private function setPageContextCredential($credential_settings) {
     foreach (SELF::$CREDENTIAL_MAPPING as $credential_key => $tag_name) {
-      $this->pageContext[$tag_name] = $credential_settings[$credential_key];
+      if (isset($credential_settings[$credential_key]) {
+        $this->pageContext[$tag_name] = $credential_settings[$credential_key];
+      }
```

Comment:
Well - the JS code treats anything in the meta tags as what should override admin values.  I don't want an empty oauth url to override and I don't want to force someone to enter the auth url everytime.